barman/output.py

```diff
@@ -676,11 +676,20 @@ def result_list_backup(self, backup_info, backup_size, wal_size, retention_statu
             self.info(backup_info.backup_id)
             return
 
-        out_list = ["%s %s " % (backup_info.server_name, backup_info.backup_id)]
+        # Backup type can be either full, incremental or rsync
+        if backup_info.backup_manager.config.backup_method == "rsync":
+            backup_type = "R"
+        elif backup_info.is_incremental:
+            backup_type = "I"
+        else:
+            backup_type = "F"
+
+        out_list = [
+            "%s %s - %s - "
+            % (backup_info.server_name, backup_info.backup_id, backup_type)
+        ]
         if backup_info.backup_name is not None:
             out_list.append("'%s' - " % backup_info.backup_name)
-        else:
```

Comment:
Why was this removed?

Reply:
We talked about this in the internal chat.
It was removed because of the way the output string is built.